include/unk_02092494.h

```diff
@@ -5,9 +5,9 @@
 #include "pokemon.h"
 #include "struct_defs/struct_02090800.h"
 
-UnkStruct_02090800 * sub_02092494(Pokemon * param0, BOOL param1, int param2);
-void sub_0209282C(UnkStruct_02090800 * param0);
+PokemonInfoDisplayStruct * sub_02092494(Pokemon * param0, BOOL param1, int param2);
+void sub_0209282C(PokemonInfoDisplayStruct * param0);
 void sub_0209304C(Pokemon * param0, TrainerInfo * param1, int param2, int param3, int param4);
-void sub_0209305C(BoxPokemon * param0, TrainerInfo * param1, int param2, int param3, int param4);
+void ProcessBoxPokemonWithTrainerInfo(BoxPokemon * param0, TrainerInfo * param1, int param2, int param3, int param4);
```

Comment:
What does this function do? `Process` is a bit generic.

Reply:
I updated it, it does kind of a bunch with cases but basically sets or processes the setting of box pokemon trainer info and met location